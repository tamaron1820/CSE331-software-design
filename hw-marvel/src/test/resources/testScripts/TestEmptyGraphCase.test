# Test a graph if loaded csv file is empty

LoadGraph heros emptyfile.csv