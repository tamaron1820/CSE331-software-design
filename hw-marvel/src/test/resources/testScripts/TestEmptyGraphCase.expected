# Test a graph if loaded csv file is empty

loaded graph heros